Features/mu_config_ext_dep.yaml

```diff
@@ -1,14 +1,12 @@
-
 #
-# This external dependency pulls MU_BASECORE for the Config feature package CI.
-# This should not be pulled when consuming this package as a dependency.
+# This external dependency pulls mu_feature_config for platforms to consume.
 #
 {
   "scope": "global",
   "type": "git",
   "name": "FEATURE_CONFIG",
   "var_name": "FEATURE_CONFIG_PATH",
-  "source": "https://github.com/microsoft/mu_feature_config.git",
-  "version": "aedb83bef7882353b3ee36958f23df65468db3eb",
+  "source": "https://github.com/kuqin12/mu_feature_config.git",

```

Comment:
Just keep it as a stub that we need to change this once we finalize the config repo update.